### PR TITLE
Chem Rework Part 1

### PR DESCRIPTION
**Pull request by Light-Weave**
## Description
TL;DR: Adds a 300-unit chem dispenser to the NCR base, along with a heater and chemmaster. There's also an old inducer in the mountain Enclave ruin- oh, and you can't wrench or screwdriver them now, so the Hospital is gonna see a lot more traffic! To compensate for no longer being able to create nearly as many chems (or replace the cell with a crowbar), the NCR gets a few broc and xander seeds- but with no dedicated camp follower, they'll need to recruit one or do it themselves.

## Motivation and Context
All part of a big chem rework. This, along with the previous PRs, is designed to effectively neuter chemistry and put wasteland medicine (stimpaks and healing powder) on top, with chemistry as a supplement. In total, here's the Chem Rework Changes:

-Dispensers cannot be moved or deconstructed. They remain where they are, forever, unless destroyed.
-Dispensers cannot be used at all by people without the perk, which is currently only available to certain jobs (Scribes, Pushers, Vault Doctors, NCR Medical Officers).
- The NCR, instead of taking the 1000-unit Hospital dispenser, now gets a 300-unit dispenser. If they want to use the Hospital one, they'll have to travel there and escort the Med Officer.
- They get some broc and xander seeds, though, so it's not too painful. Just make sure to farm them.
- And if you get a team together, you can even go get the inducer and recharge it using power cells (which you can make from the lathe... uncharged).

## How Has This Been Tested?
Yup, the usual.

## Changelog (necessary)
:cl:
add: Adds a 300-unit chem dispenser and a chemlab to the NCR base.
add: Adds broc and xander seeds to the NCR base.
add: Adds an inducer to the Enclave mountain base.
tweak: Chem dispensers can no longer be moved.
/:cl:

### CODE REVIEW

**Comment by improvedname** [comment]
Another NCR buff 👎

**Comment by Light-Weave** [comment]
> NCR buff
> they no longer have access to the 1,000 unit dispenser in the hospital or the ability to recharge it

goddammit merge conflicts

**Comment by UndeadNappist** [comment]
Again, this is basically the opposite of what I asked for.

**Comment by Light-Weave** [comment]
This is what I was told to do in the staff channel by Ren. *shrug*
